src/huts/create/endpointSpec.js

```diff
@@ -160,75 +160,75 @@ describe("'create' endpoint for huts" , function() {
+        let requestErrors = {
+            errorCount: 1,
+        };
         
-//         requestErrors[priceProperty] = {
-//             code: jasmine.any(String),
-//             da: jasmine.any(String)
-//         };
+        requestErrors.dayPrices = {
+            code: jasmine.any(String),
+            da: jasmine.any(String)
+        };
         
-//         try {
-//             await endpoint(req, res);
-//         } catch(error) {
-//             actualError = error;
-//         }
-
-//         expect(actualError).toBe(null);
-//         expect(res.status).toHaveBeenCalledWith(400);
-//         expect(res.status).toHaveBeenCalledTimes(1);
-//         expect(res.send).toHaveBeenCalledTimes(1);
-//         expect(res.send).toHaveBeenCalledWith({requestErrors});
-//     });
-
-//     theoretically.it("fails if price property %s is negative",[ "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"], async function(priceProperty) {
-//         req.body.price[priceProperty] = -priceProperty;
-//         let requestErrors = {
-//             errorCount: 1,
-//         };
+        try {
+            await endpoint(req, res);
+        } catch(error) {
+            actualError = error;
+        }
+
+        expect(actualError).toBe(null);
+        expect(res.status).toHaveBeenCalledWith(400);
+        expect(res.status).toHaveBeenCalledTimes(1);
+        expect(res.send).toHaveBeenCalledTimes(1);
+        expect(res.send).toHaveBeenCalledWith({requestErrors});
+    });
+
+    theoretically.it("fails if price property %s is negative",[ "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"], async function(priceProperty) {
+        req.body.dayPrices[priceProperty] = -priceProperty;
+        let requestErrors = {
+            errorCount: 1,
+        };
         
-//         requestErrors[priceProperty] = {
-//             code: jasmine.any(String),
-//             da: jasmine.any(String)
-//         };
+        requestErrors.dayPrices = {
+            code: jasmine.any(String),
+            da: jasmine.any(String)
+        };
         
-//         try {
-//             await endpoint(req, res);
-//         } catch(error) {
-//             actualError = error;
-//         }
-
-//         expect(actualError).toBe(null);
-//         expect(res.status).toHaveBeenCalledWith(400);
-//         expect(res.status).toHaveBeenCalledTimes(1);
-//         expect(res.send).toHaveBeenCalledTimes(1);
-//         expect(res.send).toHaveBeenCalledWith({requestErrors});
-//     });
+        try {
+            await endpoint(req, res);
+        } catch(error) {
+            actualError = error;
+        }
+
+        expect(actualError).toBe(null);
+        expect(res.status).toHaveBeenCalledWith(400);
+        expect(res.status).toHaveBeenCalledTimes(1);
+        expect(res.send).toHaveBeenCalledTimes(1);
+        expect(res.send).toHaveBeenCalledWith({requestErrors});
+    });
```

Comment:
Why is this test commented out? It seems like it would make sense to test for this?

Reply:
I don't see that it is commented out?

Reply:
Odd! :smile: